prime-finder.js

```diff
@@ -1,3 +1,25 @@
 module.exports = function(n) {
-  // do work here
+  function isPrime(candidate) {
+    // loop up to sqrt of a number to determine whether it's prime
+    for (i = 2; i ** 2 <= candidate; i++) {
```

Comment:
I think an easier way to increment from 2 until the square root of the candidate would have been:
```
i <= Math.sqrt(candidate)
```